ompi/communicator/communicator.h

```diff
@@ -233,7 +234,7 @@ typedef struct ompi_communicator_t ompi_communicator_t;
  * This is done so we don't end up needing a structure that is
  * incredibly larger than necessary because of the bitness.
  */
-#define PREDEFINED_COMMUNICATOR_PAD (sizeof(void*) * 192)
+#define PREDEFINED_COMMUNICATOR_PAD (sizeof(void*) * 194)
```

Comment:
Please add a comment of where 192 / 194 came from.  I presume you incremented by two since you added two pointers to struct mca_coll_base_comm_coll_t ?